helm/templates/deployment.yaml

```diff
@@ -2,35 +2,59 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   labels:
-    app.component: {{ .Values.containers.service.name }}
-  name: {{ .Values.containers.service.name }}
+    app: {{ .Values.name }}
+    environment: {{ .Values.environment }}
 spec:
   replicas: {{ .Values.replicas }}
```

Comment:
this field should be renamed replicaCount in in line with the name in the values file.

Reply:
Variable was named replicas in values and matched Helm chart, however have changed to replicaCount to be consistent with other services.